products/BellHybrid/services/db/ServiceDB.cpp

```diff
@@ -98,3 +127,12 @@ sys::ReturnCodes ServiceDB::InitHandler()
 
     return sys::ReturnCodes::Success;
 }
+
+bool ServiceDB::StoreIntoSyncPackage(const std::filesystem::path &syncPackagePath)
+{
+    if (!quotesDB->storeIntoFile(syncPackagePath / std::filesystem::path(quotesDB->getName()).filename())) {
```

Comment:
```suggestion
    const auto &path = syncPackagePath / std::filesystem::path(quotesDB->getName()).filename();
    if (!quotesDB->storeIntoFile(path)) {
```

That way it's easier to read